Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:12.14.0-alpine
+RUN apk add --no-cache git
+
+WORKDIR /court-backend
+
+# copy root package and lerna json files
+ADD ./package.json /court-backend/package.json
```

Comment:
Curious: any particular reason to use `ADD` instead of `COPY`?

Reply:
Actually no, based on the documentation it seems like `ADD` supports URLs, but we are not using that. I can change it, no strong opinion

Reply:
ADD also can do some "magic" like automatically extracting zip files, so better to use COPY to avoid surprises